<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sneaker Informer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pics/icon2.png</normaloff>pics/icon2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="adidasButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/adidas.png</normaloff>pics/adidas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nikeButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/nike.png</normaloff>pics/nike.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="vansButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/vans.png</normaloff>pics/vans.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="balenButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-top-color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>pics/balenciaga.png</normaloff>pics/balenciaga.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="allButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>All Shoes</string>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="inputWidget" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>221</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>In a year full of hyped releases, these are the most wanted sneakers of 2018 from our community.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>THE GREATEST DROPS OF THE YEAR.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2018'S GREATEST</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
